src/components/AddressInputPanel/index.tsx

```diff
@@ -98,7 +101,11 @@ export default function AddressInputPanel({
         <InputContainer>
           <AutoColumn gap="md">
             <RowBetween>
-              <TYPE.black color={theme.text2} fontWeight={500} fontSize={14}>
+              <TYPE.black
+                // color={theme.text2}
```

Comment:
Disabling the color prop, so we can control it exclusively with CSS.

Reply:
mmm i know its just one change but this may cause headaches in the future as it's not clear this was modded. I would rework this in pure CSS maybe with specificity? `&&&&&&` or sth idk if that'll work